pkg/api/agentPoolOnlyApi/v20180331/validate_test.go

```diff
@@ -646,7 +646,7 @@ func TestValidateAADProfile(t *testing.T) {
 				Properties: &Properties{
 					EnableRBAC: helpers.PointerToBool(true),
 					AADProfile: &AADProfile{
-						ServerAppID: "ccbfaea3-7312-497e-81d9-9ad9b8a99853",
+						ServerAppSecret: "ccbfaea3-7312-497e-81d9-9ad9b8a99853",
```

Comment:
How did this test ever succeed? Is it because before this fix these tests were never being run?

Reply:
yes :'(